### PR TITLE
feature/qp 35 assitant api client interface-package-json

### PR DESCRIPTION
**Pull request by LoganGore**
Added package.json and post-robot dependency.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "name": "@maana-io/q-assistant-client",
+  "version": "3.2.1",
+  "description": "JavaScript package to streamline communication between an assistant and the Maana Q Assistant API.",
+  "main": "AssistantAPIClient.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/maana-io/q-assistant-client.git"
```
Comment (bzvestey) [inline package.json]: This should not be using the git clone url, it should be `https://github.com/maana-io/q-assistant-client`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@maana-io/q-assistant-client",
```
Comment (bzvestey) [inline package.json]: this should be `@io-maana/q-assistant-client` as that is how it is defined in npmjs.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "version": "3.2.1",
```
Comment (bzvestey) [inline package.json]: this should be `3.2.1-beta.1` to keep it consistent with what we are doing.

File: package.json (hunk 1 above)
Comment (LoganGore) [reply package.json]: pushed

File: package.json (hunk 2 above)
Comment (LoganGore) [reply package.json]: pushed

File: package.json (hunk 3 above)
Comment (LoganGore) [reply package.json]: pushed

**Pull request merged**